dotnet/build-packages.sh

```diff
@@ -1,8 +1,8 @@
 
-RELEASE_VERSION="1.9.1"
-DEVELOPMENT_VERSION="0.1.52-prerelease"
+RELEASE_VERSION="1.9.2"
```

Comment:
We already have a version 1.9.2 that was just posted: https://www.nuget.org/packages/Datadog.AzureAppServices.DotNet/ . Was that the same deliverable as this?

Reply:
Yes I've uploaded it as I thought there shouldn't be any perf issues. 
I'm following this doc: https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2263679666/Azure+App+Services+Site+Extension+.NET+Tracer#Release-process
Where it states that we merge after the upload

Reply:
Oh, I think we usually upload the DevelopmentVerification.DdDotNet.Apm dev package while the PR is active, and then once the PR is merged we upload the Datadog.AzureAppServices.DotNet official package

Reply:
Yes, usually we merge and then get the nuget package from master